covalent_dispatcher/_core/execution.py

```diff
@@ -756,7 +863,24 @@ async def _run_planned_workflow(result_object: Result) -> Result:
 
     app_log.debug("8: All tasks finished running (run_planned_workflow)")
 
-    result_object = await _postprocess_workflow(result_object)
+    # conditional/optional post-processing of the result object
+    if result_object.lattice.get_metadata("postprocess"):
```

Comment:
This block goes in dispatcher.py -- but note the pattern used there (`runner.postprocess_workflow(dispatch_id)`).